README.md

```diff
@@ -1,2 +1,4 @@
 # git-demo
 to demonstrate how to use git and GitHub
+alejandra munoz garcia
+https://github.com/alejanmg
```

Comment:
Also, please make your name into a link that goes to that URL. The URL should not be visible.

Reply:
@alejanmg have you had a chance to look at this again?